html/images/os/ascom.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
110bytes smaller :)
```xml
<svg viewBox="0 0 283.5 58.1" xmlns="http://www.w3.org/2000/svg"><defs/><g fill="#ED1C24"><path d="M37.2 56.7c-.5-2.8-.5-5.5-.5-8.2h-.2c-3.3 6-10.3 9.6-17 9.6C9.7 58.1 0 52.3 0 41.6c0-20.3 21.8-19 36.1-19 0-8.8-3.9-12-12.5-12-6.1 0-12.2 2.4-17.1 6L6 4.2C11.7 1.6 19.1 0 25.3 0c16.6 0 24 6.6 24 23.4v33.3H37.2zM29.9 31.1c-6.4 0-16.1 1.1-16.1 9.6 0 5.2 4.7 6.8 9.1 6.8 9.6 0 13.2-8 13.2-16.4H29.9zM69.6 58.1c-5.6 0-11.2-.9-16.5-2.4l.9-11.9c4.4 2.2 9.2 3.7 14.1 3.7 4.2 0 9.5-1.8 9.5-6.8 0-10.8-25.1-3.5-25.1-23.5.1-12.2 11.1-17.2 21.8-17.2 5.4 0 10.5.9 15.8 2l-.9 11c-4.6-1.7-8.1-2.4-13-2.4-3.9 0-8.5 1.5-8.5 6.1 0 8.8 25.1 3.2 25.1 23.3 0 13-11.7 18.1-23.2 18.1zM123.4 58.1c-17.9 0-27.3-12.2-27.3-29.3.1-16.4 9.8-28.8 26.9-28.8 4.7 0 9.4.5 14 1.9l-1.3 11.4c-3.2-1.5-6.3-2.1-9.8-2.1-10.1 0-14.6 8.6-14.6 17.6 0 9.5 4.3 18.1 14.9 18.1 3.6 0 7.8-1.1 11.1-2.6l.6 11.9c-4.7 1.5-9.6 1.9-14.5 1.9zM167.8 58.1c-17.1 0-29-11.5-29-28.7 0-17.3 11.6-29.4 29-29.4s29.1 12 29.1 29.4c0 17.3-12 28.7-29.1 28.7zm0-46.9c-9.4 0-13.9 8.1-13.9 16.6 0 9.2 3 19.1 13.9 19.1 11.1 0 14-9.8 14-19.1 0-8.6-4.6-16.6-14-16.6zM269 56.7v-33c0-6-.4-12.6-8.3-12.6-9.6 0-10.8 11.5-10.8 18.7v26.9h-14.4v-33c0-6.1-.4-12.6-8.3-12.6-9.6 0-10.8 11.5-10.8 18.7v26.9h-14.4v-55.4h13.5v7.7h.2c3.7-5.6 9.7-9 16.3-9 7.5 0 13.2 2.8 16.5 9.7 3.2-6.4 10.4-9.7 17.4-9.7 13.8 0 17.5 9.7 17.5 21.8v35h-14.4z"/></g></svg>
```

Reply:
rhooo, I trusted svgminify :) What did you use to get this result ?

Reply:
https://jakearchibald.github.io/svgomg/

Reply:
Ok, bookmarked :)